packages/data/src/settings/selectors.js

```diff
@@ -44,7 +44,7 @@ export const getIsDirty = ( state, group, keys = [] ) => {
 export const getSettingsForGroup = ( state, group, keys ) => {
 	const allSettings = getSettings( state, group );
 	return keys.reduce( ( accumulator, key ) => {
-		accumulator[ key ] = allSettings[ key ] || null;
+		accumulator[ key ] = allSettings[ key ];
```

Comment:
I also [mentioned](https://github.com/woocommerce/woocommerce-admin/pull/3574#pullrequestreview-343602366) this previously, but it prevents application of defaults on destructuring when a value is null instead of undefined.

```js
const { thing = 123 } = { thing: null };
thing // null
```
Its useful for this:

```js
const { stockStatuses = {} } = useSettings( 'wc_admin', [ 'stockStatuses' ] );
```